deployment/uberspace-ansible/files/localsettings.j2

```diff
@@ -0,0 +1,17 @@
+ALLOWED_HOSTS = ['{{ url }}']
+DEBUG = False
+
+DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.mysql',
```

Comment:
```suggestion
            'ENGINE': '{{ db_backend }}',
```

It would be better to have the backend configurable. Maybe use django.db.backends.mysql as the default var for the role

Reply:
I don't think that's necessary at the moment.